controllers/middleware.js

```diff
@@ -0,0 +1,31 @@
+const { User } = require("../routes/schema");
+const { authToken } = require("../routes/auth");
+
+const middleware = async (req, res, next) => {
+  const auth = req.headers.authorization;
+  if (!auth) {
+    return res.json({
+      status: "error",
+      code: 401,
+      message: "Token is empty",
+    });
+  }
+  const user = await authToken(auth);
+  const checkUser = await User.findOne({
```

Comment:
jak authToken znalazł user to ponowne robienie tego samego tutaj w lini 14 nie ma sensu.